docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -39,7 +39,7 @@ A devfile allows to specify commands to be available for execution in a workspac
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
```

Comment:
Are all the `name:` corrections necessary? Or is it just for consistent looks?

Reply:
Looking at the specifications of the schema, here https://github.com/devfile/api/blob/master/pkg/validation/validation-rule.md 

I see that 'name' is supposed to be "limited to lowercase characters"